addons/web/static/src/legacy/js/fields/basic_fields.js

```diff
@@ -571,11 +572,74 @@ var FieldChar = InputField.extend(TranslatableFieldMixin, {
     tagName: 'span',
     supportedFieldTypes: ['char'],
     isQuickEditable: true,
+    dynamicPlaceholderOptions: {
+        readonly: false,
+        needDefaultValue: true,
+        cancelOnEscape: true,
+        chainedTitle: true,
+        filter: (model) => !["one2many", "boolean", "many2many"].includes(model.type)
+    },
 
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
+
+    /**
+     * @override
+     */
+    init: function () {
+        this._super(...arguments);
+        if (this.nodeOptions && this.nodeOptions.dynamic_placeholder) {
+            // When the dynamic placeholder is active,
+            // the recordData need to be updated when `mailing_model_real` change
```

Comment:
Same here. It's missing the dot and the wrap should be at 80 characeters, not before or after. I strongly suggest you invest a little time in finding an extension for your editor that does the rewrap of your current comment on demand with a shortcut. That way it's very flexible in the sense that it does not get in the way when you don't want to use it. If you do not like to handle word wrapping at 80 characters manually by yourself, then it will change your life.

Reply:
I'll look into it